orttraining/orttraining/test/training_ops/cuda/softmax_test.cc

```diff
@@ -55,25 +55,28 @@ static void TestSoftmaxGrad(const std::vector<int64_t>& dY_dims,
   test.CompareWithCPU(kCudaExecutionProvider, per_sample_tolerance, relative_per_sample_tolerance);
 }
 
+// TODO fix flaky test
```

Comment:
If this comment is enough to don't forget about this test while keeping a better coverage, I am ok with it.

Reply:
if we are increasing tolerances to mitigate this, the original tolerances that these random seed values cause failures for should be mentioned

Reply:
I'm not increasing the tolerance anymore.